packages.ini

```diff
@@ -611,6 +611,8 @@ python_versions = <3.12
 [jmespath==0.10.0]
 [jmespath==1.0.1]
 
+[johen==0.1.2]
```

Comment:
should probably just use `hypothesis` instead of this -- it appears to be a ~zero star version of similar things whereas hypothesis is the established property-testing library